config/models.php

```diff
@@ -279,10 +279,26 @@
         |
         */
 
-        'except'       => [
+        'except' => [
             'migrations',
         ],
 
+        /*
+        |--------------------------------------------------------------------------
+        | Specified Tables
+        |--------------------------------------------------------------------------
+        |
+        | When performing the generation of models you may want to take some of
+        | them, because you want a model for them or any other reason.
+        | You can define those tables bellow. The migrations table was
+        | filled for you, since you may not want a model for it.
```

Comment:
Would you mind updating the comments, please? :)

Reply:
I updated the description, but the problem is that I do not know English. Therefore, when translating I use Google Translator. He often incorrectly translated into other languages :(